src/main/fc/fc_tasks.c

```diff
@@ -340,7 +340,7 @@ void fcTasksInit(void)
     setTaskEnabled(TASK_CAMCTRL, true);
 #endif
 #ifdef USE_RCDEVICE
-    setTaskEnabled(TASK_RCDEVICE, rcdeviceIsEnabled());
+    setTaskEnabled(TASK_RCDEVICE, true);
```

Comment:
Not sure if this is the right thing to do - I don't think we want to waste cycles for the RC device task if there isn't even an UART configured for the RC device.

Reply:
yes, so add a condition to there: at least one uart port has Runcam device identifier?

Reply:
`rcdeviceIsEnabled()` does exactly that now.